pkg/conversion/converter.go

```diff
@@ -495,15 +495,6 @@ func (c *Converter) callCustom(sv, dv, custom reflect.Value, scope *scope) error
 // one is registered.
 func (c *Converter) convert(sv, dv reflect.Value, scope *scope) error {
 	dt, st := dv.Type(), sv.Type()
-	// Apply default values.
-	if fv, ok := c.defaultingFuncs[st]; ok {
```

Comment:
@thockin @bgrant0607 @yujuhong let's at least do a spot-check before we merge this, it is making me pretty nervous. Maybe we could check the container limit/request defaulting behavior works as expected for both pods and pod templates?